content/en/account_management/users/_index.md

```diff
@@ -56,7 +56,7 @@ To discover all of the roles available and how to create custom ones, see the [R
 
 ## Edit a user's login methods
 
-Only users with the User Access Management permission, such as users with the Datadog Admin Role, can change another user's login methods.
+Only users with the Org Management permission, such as users with the Datadog Admin Role, can change another user's login methods.
```

Comment:
Not sure what I put on the spreadsheet, but the situation seems to be:

1. Updating the login method for an _org_ requires Org Management (https://github.com/DataDog/dogweb/blob/670162dbc75ac85a966992679d8d660d6f0c7c01/dogweb/config/routing/identity.py#L35)
2. Updating the login method for a user requires User Access Manage (https://github.com/DataDog/dogweb/blob/670162dbc75ac85a966992679d8d660d6f0c7c01/dogweb/config/routing/identity.py#L61)

So this particular file can stay as-is.